astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -2293,6 +2344,39 @@ def test_linalg_diagonal(self):
         def test_linalg_trace(self):
             self.check(np.trace)
 
+        @needs_array_function
+        def test_linalg_cross(self):
+            q1 = np.array([1, 2, 3]) << u.m
+            q2 = np.array([4, 5, 6]) << u.s
+            assert_array_equal(np.linalg.cross(q1, q2), np.cross(q1, q2))
```

Comment:
Maybe add a test with `q1` and `a2 = np.array([4, 5, 6])` (no unit)

Reply:
done !